README.md

```diff
@@ -203,6 +198,24 @@ EXAMPLES
 
 _See code: [src/commands/api/update.js](https://github.com/SmartBear/swaggerhub-cli/blob/v0.2.9/src/commands/api/update.js)_
 
+## `swaggerhub api:validate`
+
+Describe the command here
+
+```
+USAGE
+  $ swaggerhub api:validate
+
+OPTIONS
+  -n, --name=name  name to print
```

Comment:
Could you try to generate the README again (`oclif-dev readme`)? The command you added doesn't have a `name` options, also the table of contents is missing.
Also could you manually remove the `OWNER/API_NAME/[VERSION]` for the commands where its been added?